### ISSUE TITLE
Support output as Directory

### ISSUE THREAD

**Issue by dangthaison91** [uber/mockolo#105]
Input params `-s --souredirs` support scan all files in a folder/directory but the output will be merged into one file.

Could you support to generate into multiple files, map 1-1 with input files?

**Comment by elsh** [comment]
Yes, it's been in the backlog.  We can do per source directory.  

If you need output per source file, can you describe a situation where that would be useful vs. per source dir?

**Comment by edias** [comment]
Do you guys have any update on this one?

**Comment by azatZul** [comment]
Also interested in this since for poorly modularised code a single file with all mocks might be gigantic
